src/needs-parens.js

```diff
@@ -1,8 +1,5 @@
-"use strict";
-
-const assert = require("assert");
-
-const { getPrecedence, shouldFlatten, isBitwiseOperator } = require("./util");
+import assert from "assert";
```

Comment:
If we're at the ESM transition anyway, I'd suggest that we'd use the [`node:`](https://2ality.com/2021/12/node-protocol-imports.html) prefix for all imports of built-in modules.

Reply:
Alright, I think that's absolutely fair. 👍